packages/shared/src/types/config/dev.ts

```diff
@@ -18,6 +18,19 @@ export type RequestHandler = (
 export type EnvironmentAPI = {
   [name: string]: {
     getStats: () => Promise<Stats>;
+
+    /**
+     * Load and execute stats bundle in Server.
+     *
+     * @param entryName - relate to rsbuild source.entry
+     * @returns the return of entry module.
+     */
+    loadBundle: <T>(entryName: string) => Promise<T>;
```

Comment:
Does this means user need to specify the return type of `loadBundle`?

Reply:
add default unknown.